src/pytorch_lightning/tuner/tuning.py

```diff
@@ -296,8 +296,19 @@ def _check_tuner_configuration(
                 " arguments should be None, please consider setting `dataloaders` instead."
             )
 
-    if any(isinstance(cb, (BatchSizeFinder, LearningRateFinder)) for cb in trainer.callbacks):
+    configured_cb = list()
+    for cb in trainer.callbacks:
+        if isinstance(cb, BatchSizeFinder) and trainer.auto_scale_batch_size:
+            configured_cb.append("BatchSizeFinder")
+        elif isinstance(cb, LearningRateFinder) and trainer.auto_lr_find:
+            configured_cb.append("LearningRateFinder")
+    if len(configured_cb) == 1:
         raise MisconfigurationException(
-            "Trainer is already configured with a `BatchSizeFinder` callback. Please remove it if you"
+            f"Trainer is already configured with a `{configured_cb[0]}` callback. Please remove it if you"
             " want to use tuner."
         )
+    elif len(configured_cb) == 2:
+        raise MisconfigurationException(
+            f"Trainer is already configured with `LearningRateFinder` and `BatchSizeFinder` callbacks."
+            " Please remove it if you want to use tuner."
```

Comment:
While we are at it, let's make it a full sentence (also above):
```suggestion
            f"Trainer is already configured with `LearningRateFinder` and `BatchSizeFinder` callbacks."
            " Please remove it if you want to use the Tuner."
```

Reply:
Also, the "please remove it" should probably become "please remove them" in the case of multiple callbacks.